packages/react/runtime/src/internal.ts

```diff
@@ -2,15 +2,14 @@
 // Licensed under the Apache License Version 2.0 that can be found in the
 // LICENSE file in the root directory of this source tree.
 
-import { createElement, lazy } from 'preact/compat';
+import { Suspense, createElement, lazy } from 'preact/compat';
```

Comment:
If we use `Suspense` from `preact`, will the `Suspense` instance in component is polyfill different?